cla_backend/libs/eligibility_calculator/cfe_civil/income.py

```diff
@@ -15,14 +15,15 @@ def translate_income(income_data):
     regular_transactions = []
 
     for income_category in INCOME_CATEGORY_TO_REGULAR_TRANSACTION:
-        amount_pence = getattr(income_data, income_category, None)
-        if amount_pence:
-            regular_transactions.append(
-                {
-                    "category": INCOME_CATEGORY_TO_REGULAR_TRANSACTION[income_category],
-                    "operation": "credit",
-                    "frequency": "monthly",
-                    "amount": pence_to_pounds(amount_pence)
-                }
-            )
+        if hasattr(income_data, income_category):
```

Comment:
We agree that we don't know why this line is necessary, but it makes the behave tests pass for an unknown reason...